apps/common/controllers/add-another-address-loop.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const _ = require('lodash');
+const BaseController = require('./base');
+
+module.exports = class AddAnotherAddressController extends BaseController {
+  constructor(options) {
+    super(options);
+    if (options.locals.field) {
+      this.field = options.locals.field;
+    } else {
+      // eslint-disable-next-line no-console
+      console.error('No field set in locals');
```

Comment:
It would be better IMO - if field is _required_ for the controller to work - throwing a big fat error would be preferable to a piddly console statement @JoeChapman @easternbloc what say you?